keps/sig-apps/85-Graduate-PDB-to-Stable/README.md

```diff
@@ -0,0 +1,318 @@
+  Not applicable, given we're missing framework allowing switching feature-gates during e2e.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  - If the leader election for the controller-manager fails, we can end up with two instances which will fight over the same PDB resources
+  - A panic in the disruption controller will crash the controller-manager
+  - Conversion between the v1beta1 and v1 versions of PodDisruptionBudget might cause problems.
+
+* **What specific metrics should inform a rollback?**
+  - Unexpected controller-manager crashes
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Manual test is planned once the implementation is finished.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  No
+
+### Monitoring Requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+  - Check number of PDBs in cluster with `etcd_object_counts{resource="poddisruptionbudgets.policy"}`
+  - Queue-related metrics to make sure the controller is functioning.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+the health of the service?**
+  - [x] Metrics
+    - Metric name: `workqueue_depth`
+    - Metric name: `workqueue_retries`
+    - Metric name: `workqueue_adds_total`
+    - Components exposing the metric: `kube-controller-manager`
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  99th percentile of `workqueue_depth` <= X per cluster-day.
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+  No
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+  No
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?** 
+  No
+
+* **Will enabling / using this feature result in introducing new API types?**
+  Graduating PDB API to GA. PodDisruptionBudget in `policy/v1`
+
+* **Will enabling / using this feature result in any new calls to the cloud 
+provider?**
+  No
+
+* **Will enabling / using this feature result in increasing size or count of 
+the existing API objects?**
+  No
+
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+  No
+
+* **Will enabling / using this feature result in non-negligible increase of 
+resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  No
+
+### Troubleshooting
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+  The controller will not work.
```

Comment:
PDB controller won't work, but Pods won't be evicted anyways because the API server is unavailable.

Reply:
Updated.